src/views/landing.njk

```diff
@@ -29,7 +29,8 @@
               isStartButton: true,
               attributes: {
                id: 'start-now',
-               onclick: '_paq.push(["trackGoal", 1]);' if piwik
+               'data-event-id': 'start-now-button' if piwik, 
+               onclick: '_paq.push(["trackGoal", todo-some-goal-id-here]);' if piwik
```

Comment:
Still awaiting the performance team to create this goal in Matomo, and then we can provide the goal ID here.